mechanics/mechanics.py

```diff
@@ -1,9 +1,37 @@
-from random import randint
+from random import choice, uniform
 
 from model.Unit import Unit
 
 golden = (1 + 5 ** 0.5) / 2
 
 
 def generate_unit(round: int) -> Unit:
-    return Unit(price=randint(int(1.5 ** round), int(3 * 1.5 ** round)) * 100)
+    # profession = choice(['magician', 'dps', 'tank', 'sniper'])
+    profession = 'sniper'
+
+    def gen(min, max, multiplier: float = 1.0) -> int:
+        return int(uniform(min, max) * multiplier ** (round - 1))
+
+    if profession == 'sniper':
```

Comment:
A lot of magic numbers, but I think they are not final.

Reply:
yes, they are to be tuned yet